tests/pykafka/test_producer.py

```diff
@@ -36,7 +35,7 @@ def test_produce(self):
 
         # set a timeout so we don't wait forever if we break producer code
         message = self.consumer.consume()
-        self.assertTrue(message.value == payload)
+        assert message.value == payload
```

Comment:
Ah thanks, got it. Yeah that should probably have been `self.assertEqual` from the start, but `assert` is definitely an improvement already.